Sources/ArcGISToolkit/Common/UtilityAssociationsElement/AssociationsFilterResultModel.swift

```diff
@@ -39,10 +45,15 @@ final class AssociationsFilterResultsModel {
     func fetchResults() {
         let element = self.element
         task?.cancel()
-        task = Task { [weak self] in
+        task = Task { [weak self, includeEmptyFilterResults] in
             guard !Task.isCancelled, let self else { return }
             let result = await Result {
-                try await element.associationsFilterResults.filter { $0.resultCount > 0 }
+                let allResults = try await element.associationsFilterResults
+                if includeEmptyFilterResults {
+                    return allResults
+                } else {
+                    return allResults.filter { $0.resultCount > 0 }
+                }
```

Comment:
```swift
                return if includeEmptyFilterResults {
                    allResults
                } else {
                    allResults.filter { $0.resultCount > 0 }
                }
```
A Swift 5.9 [feature](https://www.hackingwithswift.com/swift/5.9/if-switch-expressions) that I saw SDK uses. Up to you.